content/docs/create-a-new-react-app.md

```diff
@@ -30,43 +30,29 @@ This is also **the easiest way to integrate React into an existing website.** Yo
 
 The React team primarily recommends these solutions:
 
-- If you're **learning React** or **creating a new [single-page](/docs/glossary.html#single-page-application) app,** use [Create React App](#create-react-app).
-- If you're building a **server-rendered website with Node.js,** try [Next.js](#nextjs).
-- If you're building a **static content-oriented website,** try [Gatsby](#gatsby).
+- If you're building a **static or server-rendered application,** use [Next.js](#nextjs).
+- If you're building a **static content-oriented website,** use [Gatsby](#gatsby).
+- If you're building a **[single-page app](/docs/glossary.html#single-page-application)**, use [Create React App](#create-react-app).
```

Comment:
In the [beta.reactjs.org](https://beta.reactjs.org/learn/start-a-new-react-project#getting-started-with-a-react-toolchain) we still recommend `create-react-app` if they are just getting started. Not sure how @gaearon feels about this? We might want to make it consistent in both sites.